includes/admin/class-wc-admin-profile.php

```diff
@@ -206,6 +215,25 @@ public function save_customer_meta_fields( $user_id ) {
 			}
 		}
 	}
+
+	/**
+	 * Get user meta for a given key, with fallbacks to core user info for pre-existing fields.
+	 *
+	 * @param int    $user_id User ID of the user being edited
+	 * @param string $key     Key for user meta field
+	 */
+	public function get_user_meta( $user_id, $key ) {
+		$value = get_user_meta( $user_id, $key, true );
+		$existing_fields = array( 'billing_first_name', 'billing_last_name' );
+		if ( ! $value && in_array( $key, $existing_fields ) ) {
+			$value = get_user_meta( $user_id, str_replace( 'billing_', '', $key ), true );
+		} else if ( ! $value && $key === 'billing_email' ) {
```

Comment:
remove the space between `else` and `if`, and reverse `$key` and `'billing_email` (https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions)